src/conditional-flow/numeric-conditions.js

```diff
@@ -1,28 +1,38 @@
 // TODO: Implement the functions below to make the tests pass
 
 // This function should return true if there are no elements in the array, false otherwise
-function isArrayEmpty (array) {
-
+function isArrayEmpty(array) {
   // TODO: write code in this function body to pass the tests
+  const arrayCheck = [];
 
+  if (array.length == arrayCheck) {
```

Comment:
I would be careful doing this. It does work, but in a little bit of a hacky way. If you wanted to compare the array to an empty array, you could do this like so:
```js
if (array === arrayCheck)
```

However, by checking the length (which will be 0) against the array, you're actually doing something like this:
```js
if (0 === [])
```
Javascript will let you do this, but a lot of languages won't. So just be careful!